.github/workflows/javascript.yml

```diff
@@ -0,0 +1,91 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2023 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: JavaScript Pipeline
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  lint:
+    name: Perform linting
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "18.12.0"
```

Comment:
This is quite specific and would need constant updating. So far we it seems to work fine in GitLab CI to always use the latest LTS version, so I guess we can do so here as well. Is `lts` supported as a version specification?

Reply:
It supports versions in NVM LTS syntax: `lts/erbium`, `lts/fermium`, `lts/*` etc, but not just plain `lts`.

Reply:
Ok, what does that mean? I haven't heard of `erbium` etc. For GitLab CI we just use the [official node Docker image](https://hub.docker.com/_/node) and that has a version named `node`. Do you know which of the above would correspond to this image?